libs/application/template-api-modules/src/lib/types/index.ts

```diff
@@ -80,3 +84,11 @@ export type AttachmentEmailTemplateGenerator = (
   fileContent: string,
   email: string,
 ) => Message
+
+export interface ExternalDataNationalRegistry {
```

Comment:
This, and `ExternalDataJuristiction`, should probably extend [DataProviderResult](https://github.com/island-is/island.is/blob/main/libs/application/core/src/types/DataProviderResult.ts#L3).